pom.xml

```diff
@@ -602,6 +657,107 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>push-docker-image</id>
+            <build>
+                <pluginManagement>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.25.0</version>
+
+                        <configuration>
+                            <registry>registry.hub.docker.com/ws1connectors</registry>
+                            <authConfig>
+                                <username>appplatform</username>
```

Comment:
Please parameterize these credentials and we can set them in Bamboo. Credentials don't belong in GitHub.

Reply:
Added ```${docker.hub.username}``` and ```${docker.hub.password}``` parameters for docker hub username and key.